pkg/s2i/img/template.go

```diff
@@ -0,0 +1,13 @@
+package img
+
+const (
+	tensorflowTemplate = `FROM tensorflow/tensorflow:1.10.1-py3
```

Comment:
i'm not sure it will always work; most of the time, we need a lot libraries other than tensorflow. Do you have any solution in mind?  I think this is not specific to ciao, but a general question.

Reply:
In the current stage, I prefer all-in-one support.

In the future, I think we should support adding new layers in the image to add dependencies while I have no detailed design for it since it is out of the scope of ciao now.

Reply:
rough approaches:
1. analyze python source code and install imported packages, or use `%package xxx` syntax (slow)
2. wrap add all commonly used packages in a base image (huge image)
3. others..

Reply:
I created an issue for it. https://github.com/caicloud/ciao/issues/35